### PR TITLE
Suduku Checker

### PR DESCRIPTION
**Pull request by HiraIjaz**

### CODE REVIEW

File: 3_Suduku Checker.py [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+              ['b', 'c', 'a'],
+              ['c', 'a', 'b']]
+
+incorrect5 = [[1, 1.5],
+              [1.5, 1]]
+
+matrix6 = [[1, 1.5, 3], [3, 1, 1.5], [1.5, 3, 1]]
+
+import numpy as np
+def check_sudoku(matrix):
```
Comment (tasawernawaz) [inline 3_Suduku Checker.py:63]: there must be two blank lines after import statement

Comment (tasawernawaz) [reply 3_Suduku Checker.py:63]: can we do this without numpy as its pretty heavy library used for a simple problem.

File: 3_Suduku Checker.py [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    for i in range(size):
+      mat=np.array(matrix)
+      col=mat[:,i]
+      if all( [ isinstance(x,int) for x in row])== False:
+        return False
+      elif max(col)>size:
+        return False
+      elif len(np.unique(col))!=size :
+             return False
+    return True
```
Comment (tasawernawaz) [inline 3_Suduku Checker.py:78]: please rebase this branch with master, as there are a lot of issues related to style and formatting.

**Review by tasawernawaz — CHANGES_REQUESTED** [review]

File: 3_Suduku Checker.py (hunk 1 above)
Comment (HiraIjaz) [reply 3_Suduku Checker.py:63]: it can be solved without numpy, do you want me to solve this problem without using it?

Comment (tasawernawaz) [reply 3_Suduku Checker.py:63]: yes

Comment (HiraIjaz) [reply 3_Suduku Checker.py:63]: done

**Review by tasawernawaz — APPROVED** [review]

**Pull request merged**